os/pm/pm_procfs.c

```diff
@@ -230,7 +230,7 @@ static int power_find_dirref(FAR const char *relpath, FAR struct power_dir_s *di
 		}
 		return ERROR;
 	}
-	str = relpath;
+	str = (char *)relpath;
```

Comment:
This is not a proper fix. When relpath is declared as a `const char *` in the function prototype, it means that it is a pointer to a const string which is not supposed to be modified inside this function. I can see that `str` is also not used to modify any string value. So, better declare `str` as `const char *str`.

Reply:
Done